exchanges/auth/src/authExchange.ts

```diff
@@ -310,7 +311,9 @@ export function authExchange(
       const opsWithAuth$ = pipe(
         merge([retries.source, pendingOps$]),
         map(operation => {
-          if (bypassQueue.has(operation)) {
+          if (operation.context.authAttempt) {
```

Comment:
Worth adding [this test](https://github.com/urql-graphql/urql/discussions/3111#discussioncomment-5488476) case as well?

Reply:
Got it in https://github.com/urql-graphql/urql/pull/3112/commits/dcd44f9dd9eb5f49bd4a1daadbe3b72e055af5c0 was harder than anticipated cause I missed the result mocking :joy: